kernel/tests/write.rs

```diff
@@ -251,3 +270,382 @@ async fn test_invalid_commit_info() -> Result<(), Box<dyn std::error::Error>> {
     ));
     Ok(())
 }
+
+// check that the timestamps in commit_info and add actions are within 10s of SystemTime::now()
```

Comment:
why do we care, sorry?

Reply:
this was just a rough sanity check that we aren't wayy off. (otherwise we don't check the time field at all)

@nicklan actually suggested this and it caught a bug where one of the file's timestamps was accidentally in seconds instead of milliseconds!